azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -359,6 +361,8 @@ public void fillExecutableFromMapObject(
 
     this.setLocked(flowObj.getBool(IS_LOCKED_PARAM, false));
     this.setFlowLockErrorMessage(flowObj.getString(FLOW_LOCK_ERROR_MESSAGE_PARAM, null));
+    // Dispatch Method default is POLL i.e. 1
+    this.setDispatchMethod(DispatchMethod.fromNumVal(flowObj.getInt(FLOW_DISPATCH_METHOD, 1)));
```

Comment:
For default, you DispatchMedhod.POLL.getNumValue or something. You should not use number any where other than enum. Better to use get method. It will keep it clean.

Reply:
done